app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
You can use tuple unpacking to make your code more readable and to avoid accessing values by index. First, you should check the length of parts to ensure it's 3, and then you can unpack the values into meaningful variable names.

Reply:
And then you can use variable instead of indexing in all places